doc/api.md

```diff
@@ -955,6 +963,33 @@ The IOTA processes the entity attributes looking for a `TimeInstant` attribute.
 adds a `TimeInstant` attribute as metadata for every other attribute in the same request. With NGSI-LD, the Standard
 `observedAt` property-of-a-property is used instead.
 
+## Commands execution
```

Comment:
This section is new. We haven't talked about commands before in the documentation?

(Just to check we are not doing duplications...)

Reply:
In addition check command-related sections in https://thinking-cities.readthedocs.io/en/latest/device_api.html

Reply:
Added here: https://github.com/telefonicaid/iotagent-node-lib/pull/1551/commits/6ae6b0bece96bd7920b3b2abfc7a136041af8ad7